pilot/pkg/features/pilot.go

```diff
@@ -302,6 +302,9 @@ var (
 	EnableCAServer = env.RegisterBoolVar("ENABLE_CA_SERVER", true,
 		"If this is set to false, will not create CA server in istiod.").Get()
 
+	EnableDebugOnHTTPAddr = env.RegisterBoolVar("ENABLE_DEBUG_ON_HTTP_ADDR", true,
```

Comment:
How about: enable debug ? If off, the debug interface will be off, recommended for production.

Reply:
Or "enable_debug_on_http" - we'll expose the same functionality on the XDS-over-TLS.

Reply:
"ENABLE_DEBUG_ON_HTTP" sounds good